### PR TITLE
Feature/ui test investigation

### PR DESCRIPTION
**Pull request by achocron**
The default output of the CLI is unreadable in bitrise so I removed it

### CODE REVIEW

File: Apps/Contoso.Forms.Puppet/Contoso.Forms.Puppet/ModulePages/CrashesContentPage.xaml.cs [hunk 1]
```diff
@@ -3,7 +3,7 @@
 using Microsoft.Azure.Mobile;
 using Microsoft.Azure.Mobile.Crashes;
 using Xamarin.Forms;
-
+using System.Linq;
```
Comment (guperrot) [inline Apps/Contoso.Forms.Puppet/Contoso.Forms.Puppet/ModulePages/CrashesContentPage.xaml.cs]: Are you using that?

File: Apps/Contoso.Forms.Puppet/Contoso.Forms.Puppet/ModulePages/CrashesContentPage.xaml.cs [hunk 2]
```diff
@@ -31,7 +31,7 @@ void TestCrash(object sender, EventArgs e)
 
         void DivideByZero(object sender, EventArgs e)
         {
-            (42 / int.Parse("0")).ToString();
+            int x = (42 / int.Parse("0"));
```
Comment (guperrot) [inline Apps/Contoso.Forms.Puppet/Contoso.Forms.Puppet/ModulePages/CrashesContentPage.xaml.cs]: Might want to suppress warning of unused local variable.

File: Tests/Contoso.Forms.Test.Strings/Strings.cs [hunk 3]
```diff
@@ -0,0 +1,96 @@
+﻿using System;
```
Comment (guperrot) [inline Tests/Contoso.Forms.Test.Strings/Strings.cs]: Just curious, does xamarin forms has a proper way to handle localizable strings?

File: scripts/run-ui-tests.sh [hunk 4]
```diff
@@ -0,0 +1,144 @@
+#!/bin/bash
```
Comment (guperrot) [inline scripts/run-ui-tests.sh]: Might want to filter output on bitrise.

File: Apps/Contoso.Forms.Puppet/Contoso.Forms.Puppet.Droid/Contoso.Forms.Puppet.Droid.csproj [hunk 5]
```diff
@@ -26,7 +27,8 @@
     <ErrorReport>prompt</ErrorReport>
     <WarningLevel>4</WarningLevel>
     <AndroidLinkMode>None</AndroidLinkMode>
-    <AndroidSupportedAbis>arm64-v8a;armeabi;armeabi-v7a;x86</AndroidSupportedAbis>
+<EmbedAssembliesIntoApk>true</EmbedAssembliesIntoApk>
```
Comment (guperrot) [inline Apps/Contoso.Forms.Puppet/Contoso.Forms.Puppet.Droid/Contoso.Forms.Puppet.Droid.csproj]: Wrong indent

**Comment by achocron** [comment]
@guperrot What should start new version do for the Test apps, though? Do the assembly info version attributes matter? And should they match Demo? In Puppet, we don't have them

File: Tests/Contoso.Forms.Test.Strings/Strings.cs (hunk 3 above)
Comment (guperrot) [inline Tests/Contoso.Forms.Test.Strings/Strings.cs]: Can you try?

File: Tests/Contoso.Forms.Test/Properties/AssemblyInfo.cs [hunk 6]
```diff
@@ -0,0 +1,26 @@
...
+[assembly: AssemblyProduct("")]
+[assembly: AssemblyCopyright("")]
+[assembly: AssemblyTrademark("")]
+[assembly: AssemblyCulture("")]
+
+// The assembly version has the format "{Major}.{Minor}.{Build}.{Revision}".
+// The form "{Major}.{Minor}.*" will automatically update the build and revision,
+// and "{Major}.{Minor}.{Build}.*" will update just the revision.
+
+[assembly: AssemblyVersion("1.0.*")]
```
Comment (guperrot) [inline Tests/Contoso.Forms.Test/Properties/AssemblyInfo.cs]: Can you use same assembly versions as puppet and make sure the `start-new-version.sh` updates those?

File: Tests/Contoso.Forms.Test.Strings/Strings.cs (hunk 3 above)
Comment (achocron) [inline Tests/Contoso.Forms.Test.Strings/Strings.cs]: Yes but let's do that in a new PR

**Comment by guperrot** [comment]
Now waiting on warnings fix.

**Pull request merged**